src/code/code_800F7260.c

```diff
@@ -361,101 +358,101 @@ void func_800F7CEC(u8 arg0) {
         if (gSoundBanks[arg0][phi_s4].unk_2B == 0) {
             func_800F7B54(arg0, phi_s4);
         } else if (gSoundBanks[arg0][phi_s4].unk_2A != 0) {
-            temp_s2 = &gSoundBanks[arg0][phi_s4];
+            entry = &gSoundBanks[arg0][phi_s4];
 
-            if (&D_801333D4.x == temp_s2[0].posX) {
-                temp_s2->unk_1C = 0.0f;
+            if (&D_801333D4.x == entry[0].posX) {
+                entry->dist = 0.0f;
             } else {
-                tempf1 = *temp_s2->posY * 1;
-                temp_s2->unk_1C = (SQ(*temp_s2->posX) + SQ(tempf1) + SQ(*temp_s2->posZ)) * 1;
+                tempf1 = *entry->posY * 1;
+                entry->dist = (SQ(*entry->posX) + SQ(tempf1) + SQ(*entry->posZ)) * 1;
             }
-            spA9 = temp_s2->unk_24;
-            if (temp_s2->unk_26 & 0x10) {
-                temp_s2->unk_20 = SQ(0xFF - spA9) * 0x1690;
+            spA9 = entry->unk_24;
+            if (entry->unk_26 & 0x10) {
+                entry->unk_20 = SQ(0xFF - spA9) * 0x1690;
             } else {
-                if (temp_s2->unk_1C > 0x7FFFFFD0) {
-                    temp_s2->unk_1C = 0x70000008;
-                    osSyncPrintf(D_80133344, temp_s2->unk_28, temp_s2->posX, temp_s2->posZ, *temp_s2->posX,
-                                 *temp_s2->posY, *temp_s2->posZ);
+                if (entry->dist > 0x7FFFFFD0) {
+                    entry->dist = 0x70000008;
+                    // VT_COL(RED, WHITE) "<INAGAKI CHECK> dist over! flag:%04X ptr:%08X pos:%f-%f-%f" VT_RST "\n"
```

Comment:
```suggestion
```
I assume this will be improved once the .data situation improves, in the mean time people can just look at `D_80133344`